modules/ddb_cover_upload/css/ddb_cover_upload.admin.css

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file
+ * Styling for ddb cover upload module.
+ * This should only be added to cover upload specific pages.
+ */
+
+.search-result .ting-object .group-inner {
+  float: left;
+  width: 75%;
+}
+
+.field-name-ting-abstract {
+  margin-bottom: 10px;
+}
+
+.conditions p {
+  max-width: 780px;
+}
+
+.form-item-ddb-cover-upload-confirm label.option {
+  font-weight: bold;
+  line-height: 1.5;
+  vertical-align: middle !important; /* Align input and label */
+}
+
+.form-item-ddb-cover-upload-confirm .form-checkbox {
+  line-height: 1.5;
+  vertical-align: middle;
+}
+
+.form-item-ddb-cover-upload-material .form-text {
+  max-width: 100%; /* Avoid overlapping right column */
+}
+
+.guide {
+  border: 1px solid #f5f5f5;
+  margin-bottom: 2rem;
+}
+
+.guide__row {
+  width: 100%;
+  border-radius: 2px;
+  padding: 1rem;
+  display: flex;
+}
+
+.guide__col--12 {
+  width: 100%;
+  margin: 1rem;
+}
+
+.guide__col--3 {
+  width: 33%;
+  margin: 1rem;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.img-displayed {
+  max-width: 100%;
+}
+
+.img-submitted {
+  max-width: 150px;
+}
+
+.changed-warning {
+  color: #ed541d; /* Seven color */
+  font-size: .7em;
+  font-weight: normal;
+}
+
+.right.form-wrapper h3 {
+  text-decoration: underline;
+}
+
+#edit-finish {
+  float: right;
+}
+
+.overflow-hidden {
+  overflow: hidden;
+}
+
+.btn-form-submit {
+  background: rgb(243,243,243);
+  background: linear-gradient(180deg,rgba(243,243,243,1) 0%,rgba(224,224,224,1) 100%);
+  border: 1px solid #e4e4e4;
+  border-bottom: 1px solid #b4b4b4;
+  border-left-color: #d2d2d2;
+  border-right-color: #d2d2d2;
+  border-radius: 20px;
+  background-color: #ccc;
+  padding: 4px 17px;
+  color: #5a5a5a;
+}
+
+.btn-form-submit:hover {
+  background: linear-gradient(0deg,rgba(243,243,243,1) 0%,rgba(224,224,224,1) 100%);
+  color: #222222;
+  text-decoration: none;
+}
+
+.btn-form-submit.disabled {
+  background: #eee none;
+  border-color: #eee;
+  text-shadow: none;
+  color: #999;
+}
```

Comment:
Why do we need these classes at all?

They seem like a redeclaration of [`a.button` from Seven](https://github.com/drupal/drupal/blob/7.x/themes/seven/style.css#L655).